sklearn/metrics/tests/test_pairwise.py

```diff
@@ -245,13 +254,26 @@ def callable_rbf_kernel(x, y, **kwds):
 @pytest.mark.parametrize(
         'func, metric, kwds',
         [(pairwise_distances, 'euclidean', {}),
+         (pairwise_distances, minkowski, _minkowski_kwds),
+         (pairwise_distances, 'minkowski', _minkowski_kwds),
          (pairwise_distances, wminkowski, _wminkowski_kwds),
          (pairwise_distances, 'wminkowski', _wminkowski_kwds),
          (pairwise_kernels, 'polynomial', {'degree': 1}),
          (pairwise_kernels, callable_rbf_kernel, {'gamma': .1})])
 @pytest.mark.parametrize('array_constr', [np.array, csr_matrix])
 @pytest.mark.parametrize('dtype', [np.float64, int])
 def test_pairwise_parallel(func, metric, kwds, array_constr, dtype):
+    # wminkowki metric will be deprecated starting from scipy 1.6
+    # and removed in scipy 1.8
+    if sp_version >= parse_version("1.6.0") and (
+        metric == wminkowski or metric == 'wminkowski'
+    ):
+        metric = 'minkowski'
```

Comment:
In this case, we can skip because `minkowski` is tested explicitly and this would be testing it again.

Reply:
I'm not sure to understand correctly: I have then skipped the wminkowski and 'wminkowski' test for scipy >=1.6.0, Is that what you meant? Thanks!

Reply:
> I have then skipped the wminkowski and 'wminkowski' test for scipy >=1.6.0, Is that what you meant?

Yes this is what I meant.

I think we can skip here because this PR adds `metric = 'minkowski'` to `pytest.mark.parameterize`, so `metric = 'minkowski'` will get tested when it gets to that test case.